resources/js/components/Product/AddToCart.vue

```diff
@@ -183,15 +199,45 @@ export default {
 
         productOptions: function () {
             let options = []
+            let customOptions = []
+
             Object.entries(this.options).forEach(([key, val]) => {
                 options.push({
                     option_id: key,
                     option_value: val,
                 })
             })
+
+            Object.entries(this.customOptions).forEach(([key, val]) => {
+                if (typeof val === 'string' && val.startsWith('FILE;')) {
+                    let values = val.split(';', 4)
```

Comment:
It might be nice for readability to use the destructuring assignment
```suggestion
                    let [prefix, name, type, data] = val.split(';', 4)
```